### DISCUSSION TITLE
Why do we need queue_bind in asynchronous_publisher_example.py?

### DISCUSSION THREAD

**Discussion by hmtrii** [pika/pika#1443] [Q&A]
Hi there,
I wonder do we need queue_bind method in `examples/asynchronous_publisher_example.py`? 
I think the publisher does not care about which queue name is. 
Any ideas?

**Comment by lukebakken** [comment]
It was written that way in case you start the publisher before you start the consumer. Without binding the queue to the exchange the messages would be dropped since they wouldn't be routed anywhere.
